FightPandemics/Utilities/Extensions/UINavigationBar+FightPandemics.swift

```diff
@@ -27,6 +27,24 @@
 import UIKit
 
 extension UINavigationBar {
+    /// Configures the navigation bar appearance
+    func configure() {
+        if #available(iOS 13.0, *) {
```

Comment:
we're targeting `13.0+` - no need for the conditional

Reply:
Okay, cool, I will change that